tagged_sdk_packages_list.txt

```diff
@@ -1,10 +1,12 @@
+platforms:android-34
```

Comment:
target 35 to match bubblewrap? https://github.com/GoogleChromeLabs/bubblewrap/blob/2395d6aea6be141c146a1db435bc21fd06c41978/packages/core/template_project/app/build.gradle#L59